### DISCUSSION TITLE
带宽限速问题-使用的带宽不到限制，延迟却变得非常高

### DISCUSSION THREAD

**Discussion by 2627844225** [natfrp/wiki#313] [使用问题]
我的带宽限速为36mbps，而我的宽带单线上传为900mbps,且网络环境是nat1，主机是机架式服务器双路e5/48g，系统是winserver,线路分别使用一条青铜线路和一条白银线路，我的测试结果是一样的延迟变高
我用来开方舟服务器的，单个客户端限制最大连接带宽是60kb/s，当我的客户端连接超过5个即300kb/s时游戏的延迟就到了70-80ms在网速监控上也能看到上传仅有300kb/s左右，那问题来了，我的带宽足够大，且连接需要用到的带宽也远远达不到限速，为什么延迟会随着玩家的连接数越来越大而高？理论我的限速可以承受70名玩家的稳定连接即4200kb/s，4.1mb/s 32mbps，但为什么仅仅10名左右就能变得如此之卡，这并不是单个客户端变卡了而是整体延迟变高了，所以这也不是地区问题，也并不是带宽问题，所以有什么解决方案吗？或者请告诉我为什么延迟会变高？
因为用到的带宽远远不及限制的十分之一

**Comment by fakeboboliu** [comment]
您好，

我正在尝试复现以处理这个问题。
基于您的描述，我建议您尝试在多个服务器开启多个隧道并将用户分流，
如果这个方法有效，请务必告知我，这将是一个重要的信息。

**Reply by 2627844225** [reply]
有单个服务器用两三条隧道，而我测试的结果为整体变卡了，而不是某个隧道的问题
